src/js/letters/utils/helpers.jsx

```diff
@@ -255,7 +255,8 @@ export function getBenefitOptionText(option, value, isVeteran, awardEffectiveDat
     valueString = value;
   }
 
-  const isAvailable = value && value !== AVAILABILITY_STATUSES.unavailable;
+  // NOTE: 0 is a legitimate return value
+  const isAvailable = (value === 0 || value) && value !== AVAILABILITY_STATUSES.unavailable;
```

Comment:
Wait, so this is either a status string or a value...oi. A'ight...

Reply:
Yeah to be honest I'm not even sure if it _can_ be a status string.

Reply:
Oh.
Hmm.

@kreek Do we have a swagger doc for what we can expect from `vets-api`?

Reply:
Well it's more about what we're doing with it after we get it (not sure where we might be transforming to `unavailable`). I do know that we can get back `null` or a number (including 0), and those two should obviously be treated differently.

Reply:
@cvalarida I'm considering getting rid of the `unavailable` condition. We don't modify any of the `benefitInformation` contents before sending them to the redux store, so getting a value of `unavailable` shouldn't be possible unless it's directly returned by `vets-api`. This doesn't seem to be the case looking at swagger - all of the values are booleans, nullable booleans, or numbers (also nullable, apparently, as in the case of `monthly_award_amount`, though this isn't documented). The only string type is for `award_effective_date` - not sure what the expected return values are there.

Here's how we put the response into redux: 
https://github.com/department-of-veterans-affairs/vets-website/blob/7c2ae4148ec865cb34bea6cd0007841376af87a5/src/js/letters/reducers/index.js#L95

Reply:
@cvalarida to view the swagger documentation locally follow the [directions in vets-api](https://github.com/department-of-veterans-affairs/vets-api/tree/master/app/swagger) (we're in the process of getting is easier to view online)